src/pip/_internal/req/req_uninstall.py

```diff
@@ -130,8 +132,9 @@ def compress_for_output_listing(paths):
                 if fname.endswith(".pyc"):
                     continue
 
-                file_ = os.path.normcase(os.path.join(dirpath, fname))
-                if os.path.isfile(file_) and file_ not in files:
+                file_ = os.path.join(dirpath, fname)
+                if (os.path.isfile(file_) and
```

Comment:
How about making a should_skip variable here? That way the conditional can be more readable. :)

Reply:
Assigning a should_skip variable would still have to go across two lines, so no real readability improvement. TBH, I can't be bothered - this is a case where arguing about readbility improvements is non-productive IMO, so I'd rather just leave it.

At the root, I just disagree with flake8's rule here, so closing my eyes and doing something that conforms to the rule is sufficient for me.

Reply:
Cool. I don't think it's super important anyway. :P